src/SFA.DAS.Reservations.Web/Stubs/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace SFA.DAS.Reservations.Web.Stubs
+{
+    public static class ConfigurationExtensions
+    {
+        public static bool UseStub(this IConfiguration config)
+        {
+            var useStub = config.GetValue<bool>("UseStub");
```

Comment:
given the useStub is not being used i'd just return the value without the need of a var. Its useful for debug purposes but given its just returning the value you can just debug the return value instead.